packages/web-components/fast-components/src/listbox/listbox.styles.ts

```diff
@@ -38,43 +65,56 @@ export const listboxStyles: FoundationElementTemplate<ElementStyles> = (
             padding: calc(${designUnit} * 1px) 0;
         }
 
-        :host(:focus-within:not([disabled])) {
-            border-color: ${focusStrokeOuter};
-            box-shadow: 0 0 0 calc((${focusStrokeWidth} - ${strokeWidth}) * 1px)
-                ${focusStrokeOuter} inset;
-        }
+        ${!hostContext ? css`
+            :host(:focus-within:not([disabled])) {
+                border-color: ${focusStrokeOuter};
+                box-shadow: 0 0 0
+                    calc((${focusStrokeWidth} - ${strokeWidth}) * 1px)
+                    ${focusStrokeOuter} inset;
+            }
+
+            :host([disabled]) ::slotted(*) {
+                cursor: ${disabledCursor};
+                opacity: ${disabledOpacity};
+                pointer-events: none;
+            }
+        ` : ``}
 
-        :host([disabled]) ::slotted(*) {
-            opacity: ${disabledOpacity};
-            cursor: ${disabledCursor};
-            pointer-events: none;
+        ::slotted([aria-selected="true"][aria-checked="false"]) {
+            border-color: ${neutralLayerFloating};
         }
 
-        :host([size]) {
+        ${hostContext || `:host([size])`} {
             max-height: calc(
-                (var(--size) * ${heightNumber} + ((${designUnit} + ${strokeWidth}) * 2)) *
-                    1px
+                (${listboxSize} * ${heightNumber} + var(--stroke-offset)) * 1px
```

Comment:
I see this may be related to the previous part where you're calculating the size differently. If we need to know that we can rework this to come up with a default size, essentially the padding value top and bottom plus the default content height, which for `base` text and icons will be 16px. Of course we don't want to hard-code that though.